### DISCUSSION TITLE
Single character domains (such as a.example.com) cannot be used

### DISCUSSION THREAD

**Discussion by bana118** [mozilla/hubs#4554] [Hubs Cloud Questions] [answered]
I tried to create an AWS Hubs Cloud with a domain like "a.example.com" but it failed with an error message saying "Must be a valid domain name (eg myhub.com or hub.mycompany.com)".
Oddly enough, it succeeded for the domain "a..example.com".
Regular expressions about domains are most likely to be wrong. (https://github.com/mozilla/hubs-ops/issues/117)
Will this problem be fixed?

**Comment by pattersonbl2** [comment, marked as answer]
Thank you for opening a issues on this.  i don't have a timeline when this will be reviewed and fixed due to the size of the team.

**Reply by bana118** [reply]
Thanks for the reply. I'm patiently waiting for the problem to be fixed.
